packages/components/src/Typography/css/TextColors.css

```diff
@@ -50,34 +54,218 @@
   color: var(--color-text--secondary);
 }
 
+.textReverse {
+  color: var(--color-text--reverse);
+}
+
+.textReverseSecondary {
+  color: var(--color-text--reverse--secondary);
+}
+
+/* Statuses */
+
 .inactive {
   color: var(--color-inactive--onSurface);
 }
 
+.inactiveSurface {
+  color: var(--color-inactive--surface);
+}
+
+.inactiveOnSurface {
+  color: var(--color-inactive--onSurface);
+}
+
 .critical {
   color: var(--color-critical);
 }
 
+.criticalSurface {
+  color: var(--color-critical--surface);
+}
+
+.criticalOnSurface {
+  color: var(--color-critical--onSurface);
+}
+
 .warning {
   color: var(--color-warning--onSurface);
 }
```

Comment:
It definitely _is_ weird* but similar to the [mobile PR](https://github.com/GetJobber/atlantis/pull/2033), the plan for this PR is just adding more colors to the foundational Typography components. There is a [ticket for mobile](https://jobber.atlassian.net/browse/JOB-105200) that I think we could expand the scope of to include web?

***TLDR:** `warning`, `informative`, and formerly `success` and `critical`, would fail WCAG AA contrast ratios when used in isolation on top of `surface`, so at the time I thought it would be smarter to use the `onSurface` values as the default, but now that we are adding the actual onSurface values, it is much weirder